elytron/src/main/resources/org/wildfly/extension/elytron/LocalDescriptions.properties

```diff
@@ -1139,7 +1139,10 @@ elytron.key-store.alias.certificate-chain.version=The certificate version.
 
 elytron.modifiable-key-store.read-alias=Read an alias from a KeyStore.
 elytron.modifiable-key-store.read-alias.alias=The alias of the KeyStore item to read.
+elytron.modifiable-key-store.read-alias.verbose=Instruct read to show HEX format fields content.
 elytron.modifiable-key-store.read-aliases=Read aliases from a KeyStore.
+elytron.modifiable-key-store.read-aliases.recursive=Read more than name of aliases in keystore.
```

Comment:
Maybe we could reword this to something like:

Include information about each alias in the KeyStore. The default value is false.